pynestml/utils/model_installer.py

```diff
@@ -45,13 +47,23 @@ def install_nest(target_path: str, nest_path: str) -> None:
     InvalidPathException
         If a failure occurs while trying to access the target path or the NEST installation path.
     """
+    cmake_cmd = ["cmake"]
+    if not os.path.isdir(nest_path):
+        raise InvalidPathException(f"NEST path ({nest_path}) is not a directory!")
+
+    nest_config_path = '-Dwith-nest=' + os.path.join(nest_path, 'bin', 'nest-config')
+    cmake_cmd.append(nest_config_path)
+
+    if install_path:
+        if not os.path.isabs(install_path):
```

Comment:
Could you also check if `install_path` is a directory?

Reply:
`install_path` may not exist, and it will be created by `cmake` if necessary.